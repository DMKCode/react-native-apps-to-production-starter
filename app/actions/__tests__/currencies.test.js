import {
  getInitialConversion,
  changeCurrencyAmount,
  swapCurrency,
  changeBaseCurrency,
  changeQuoteCurrency,
} from '../currencies';

it('creates a GET_INITIAL_CONVERSION action', () => {
  expect(getInitialConversion()).toMatchSnapshot();
});

it('creates a CHANGE_CURRENCY_AMOUNT action', () => {
  expect(changeCurrencyAmount(1000)).toMatchSnapshot();
});

it('creates a SWAP_CURRENCY action', () => {
  expect(swapCurrency()).toMatchSnapshot();
});

it('creates a CHANGE_BASE_CURRENCY action', () => {
  expect(changeBaseCurrency('USD')).toMatchSnapshot();
});

it('creates a CHANGE_QUOTE_CURRENCY action', () => {
  expect(changeQuoteCurrency('GBP')).toMatchSnapshot();
});
