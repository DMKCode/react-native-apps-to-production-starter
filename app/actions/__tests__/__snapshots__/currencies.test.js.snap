// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a CHANGE_BASE_CURRENCY action 1`] = `
Object {
  "currency": "USD",
  "type": "CHANGE_BASE_CURRENCY",
}
`;

exports[`creates a CHANGE_CURRENCY_AMOUNT action 1`] = `
Object {
  "amount": 1000,
  "type": "CHANGE_CURRENCY_AMOUNT",
}
`;

exports[`creates a CHANGE_QUOTE_CURRENCY action 1`] = `
Object {
  "currency": "GBP",
  "type": "CHANGE_QUOTE_CURRENCY",
}
`;

exports[`creates a GET_INITIAL_CONVERSION action 1`] = `
Object {
  "type": "GET_INITIAL_CONVERSION",
}
`;

exports[`creates a SWAP_CURRENCY action 1`] = `
Object {
  "type": "SWAP_CURRENCY",
}
`;
